src/applications/vaos/components/FacilityAddress.jsx

```diff
@@ -12,8 +16,23 @@ export default function FacilityAddress({
   showPhone = true,
   level = 4,
 }) {
+  const { id } = useParams();
+  const appointment = useSelector(
```

Comment:
I don't think this is the way we want to approach showing the phone here. This component is used throughout the app in different contexts, which is why it's in the top level components folder. This would tie the covid phone functionality to having an appointment selected from the appt list, which I think would be limiting for us, especially since we need to show the covid phone on three pages in the vaccine flow.

Reply:
I have pulled the covid specific bits out and pass in a simple prop on whether to show the vaccine line or not.